### PR TITLE
Create OmniXAI Recipe

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge/staged-recipes  ready for review

File: recipes/OmniXAI/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - tqdm
+    - wheel
+    - packaging
+    - ipython
+    - tabulate
+    - statsmodels >=0.10.1
+
+test:
+  imports:
+    - docs
```
Comment (xhochy) [inline recipes/OmniXAI/meta.yaml:44]: This sounds wrong and it looks like it is installing its docs globally. Can you open an upstream bug and add an extra step that does `rm -rf ${SP_DIR}/docs`?

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/OmniXAI/meta.yaml (hunk 1 above)
Comment (rxm7706) [reply recipes/OmniXAI/meta.yaml]: When I remove the docs from the Tests - The import fails. 

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=627207&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823


`2022-12-12T14:12:51.1479352Z export PREFIX=/home/conda/staged-recipes/build_artifacts/omnixai_1670854274661/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol
2022-12-12T14:12:51.1483172Z export SRC_DIR=/home/conda/staged-recipes/build_artifacts/omnixai_1670854274661/test_tmp
2022-12-12T14:12:52.0443550Z import: 'omnixai'
2022-12-12T14:12:52.0444845Z Traceback (most recent call last):
2022-12-12T14:12:52.0446955Z   File "/home/conda/staged-recipes/build_artifacts/omnixai_1670854274661/test_tmp/run_test.py", line 2, in <module>
2022-12-12T14:12:52.0447892Z     import omnixai
2022-12-12T14:12:52.0448484Z ModuleNotFoundError: No module named 'omnixai'
2022-12-12T14:12:53.1578823Z Tests failed for omnixai-1.2.3-pyhd8ed1ab_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
2022-12-12T14:12:53.1580894Z WARNING:conda_build.build:Tests failed for omnixai-1.2.3-pyhd8ed1ab_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
2022-12-12T14:12:53.1888425Z TESTS FAILED: omnixai-1.2.3-pyhd8ed1ab_0.conda`

File: recipes/OmniXAI/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/omnixai-{{ version }}.tar.gz
+  sha256: 2f8c9ed9ae3977b19f3fcd40a33a56b94fabc76ee63652c101dcaa39db40ba98
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+# Open Upstream Bug, looks like it is installing its docs globally
+  script: rm -rf ${SP_DIR}/docs
```
Comment (xhochy) [inline recipes/OmniXAI/meta.yaml:16]:
```suggestion
  script:
   - {{ PYTHON }} -m pip install . -vv
   # Open Upstream Bug, looks like it is installing its docs globally
   - rm -rf ${SP_DIR}/docs
```

**Review by xhochy — APPROVED** [review]
No need to get this working on `win`. This is a `noarch: python` package and will be built solely on Linux once merged.

**Pull request merged**